modules/identity-provider-provisioning-user-lookup-mapping.adoc

```diff
@@ -1,35 +1,37 @@
 // Module included in the following assemblies:
 //
-// * orphaned
+// * authentication/identity_providers/configuring-basic-authentication-identity-provider.adoc
 
 [id="identity-provider-provisioning-user-lookup-mapping_{context}"]
 = Manually provisioning a user when using the lookup mapping method
 
 When using the `lookup` mapping method, user provisioning is done by an external system, via the API.
-Typically, identities are automatically mapped to users during login. The 'lookup' mapping method automatically
+Typically, identities are automatically mapped to users during login. The `lookup` mapping method automatically
 disables this automatic mapping, which requires you to provision users manually.
 
-
-.Procedure 
+.Procedure
 
 If you are using the `lookup` mapping method, use the following steps for each user after configuring
 the identity provider:
 
 . Create an {product-title} User, if not created already:
 +
+[source,terminal]
 ----
 $ oc create user <username>
 ----
 +
 For example, the following command creates an {product-title} User `bob`:
 +
+[source,terminal]
 ----
 $ oc create user bob
 ----
 
 . Create an {product-title} Identity, if not created already. Use the name of the identity provider and
 the name that uniquely represents this identity in the scope of the identity provider:
 +
+[source,terminal]
 ----
 $ oc create identity <identity-provider>:<user-id-from-identity-provider>
```

Comment:
`the name that uniquely represents this identity in the scope of the identity provider` is correct. But `<user-id-from-identity-provider>` is not always correct. In LDAP identity provider, it is a little tricky (which needs specific doc). You can change to `$ oc create identity <identity-provider>:<name-that-uniquely-represents-the-user-in-the-identity-provider>`

Reply:
See redhat.com KB [How to create user, identity and map user and identity in LDAP authentication for 'mappingMethod' as 'lookup'](https://url.corp.redhat.com/7c27ef3 )

Reply:
And redhat.com another KB [How to create user, identity and map user and identity in OIDC authentication for 'mappingMethod' as 'lookup'](https://url.corp.redhat.com/213461e)